README.md

```diff
@@ -114,6 +116,46 @@ To stop the containers run:
 
 ## Application Concepts
 
+## Usage
+
+This API was built following REST pattern, then you can access all cruds this way:
+
+- Companies:
+
+  index: GET http://localhost:3000/companies
```

Comment:
We also can add the `http://localhost:3000` as the base URL, and then for the examples, you can use just the path (`/companies`)